utils/s2n_socket.c

```diff
@@ -165,6 +187,9 @@ int s2n_socket_read(void *io_context, uint8_t *buf, uint32_t len)
         return -1;
     }
 
+    /* Clear the quickack flag so we know to reset it */
+    ((struct s2n_socket_read_io_context*) io_context)->tcp_quickack_set = 0;
```

Comment:
Does this comment still applicable https://github.com/awslabs/s2n/blob/48c307194e08fc7446f3954b49e6a1cbe1de4bfc/tls/s2n_connection.c#L797

Reply:
yes, corked io and quickack are different.